design/one-pager-ignore-changes.md

```diff
@@ -0,0 +1,199 @@
+and then Crossplane resets it back to the `readCapacity` and `writeCapacity` defined on the Table, which effectively makes autoscaling useless.
+- [NodeGroup] while using a node autoscaler. There is a race condition between crossplane and node autoscaler. 
+Node autoscaler increases the `desiredSize` due to resource requirement but crossplane decrease it to `desiredSize` set in composition.
+
+In these cases, it would make sense to make Crossplane ignore select parameters during updates. 
+Terraform addresses this use case with its [ignore_changes] functionality. 
+
+An additional benefit of ignoring some fields would be that it would be possible to have different Crossplane
+Compositions managing a single external resource. This could open up some interesting new possibilities for Crossplane.
+
+So in this document, the aim will be to introduce a solution in Crossplane for ignoring some managed resource parameters
+during updates.
+
+## Goals
+
+* Design a functionality that allows for ignoring some fields during resource updates.
+* Try and make it as easy as possible to apply the ignoring functionality on existing Crossplane providers. In other words, to 
+design a standard way on how to update the providers code to support `ignore changes`.
+
+## Solution
+
+The idea is to introduce a new `ignoreChanges` string array field in the `spec` of the managed resource, that would work similarly
+to Terraform's [ignore_changes]. The items in the array would be the field paths of the fields that should
+be ignored on updates. 
+
+```yaml
+apiVersion: eks.aws.crossplane.io/v1alpha1
+kind: NodeGroup
+metadata:
+  name: my-group
+spec:
+  ignoreChanges:
+    - scalingConfig.desiredSize
+  forProvider:
+    region: us-east-1
+    clusterNameRef:
+      name: sample-cluster
+    subnetRefs:
+      - name: sample-subnet1
+    nodeRoleRef:
+      name: somenoderole
+    scalingConfig:
+      desiredSize: 1
+      maxSize: 5
+      minSize: 1
+    updateConfig:
+      maxUnavailablePercentage: 50
+      force: true
+  providerConfigRef:
+    name: example
+```
+
+The solution should affect just the Update part of the Crossplane lifecycle, so create and late initialization should work 
+as before, even though a field is marked under `ignoreChanges`. Additionally, as Crossplane will soon support `status.atProvider`,
+users will be able to observe the actual values of the ignored fields.
+
+#### Ignoring required fields
+
+Required fields should not be put into `ignoreChanges`, as that could compromise the functionality of the provider. If such
+a need arises, it should be handled as a special case on the provider side, and not through the `ignoreChanges` feature.
+
+Of course, some fields could be marked as required, but they are actually required only for create. As there currently is
+no standard on differentiating them, it is advised to use `ignoreChanges` on required fields only if certain that it's not
+required for update.
```

Comment:
This seems unclear to me. Are we saying that required fields cannot have their changes ignored, or not?

Reply:
My point is that users should be careful with putting required fields into `ignoreChanges` as its not clear from the resource spec if the field is required only in create or both in create and update.

 I dont expect needing some validation for that, just put a note about it into the docs